src/test/unit/mocks/magistratesPublicList.json

```diff
@@ -23,22 +23,13 @@
         {
             "courtHouse": {
```

Comment:
There's a couple examples of caseName near the bottom that need to be removed

Reply:
Hi Chris, removed the 2 examples of caseName